app/models/pageflow/entry.rb

```diff
@@ -54,6 +54,7 @@ def feature_state(name)
 
     def publish(options = {})
       ActiveRecord::Base.transaction do
+        self.first_published_at ||= Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.